src/cargo/core/compiler/mod.rs

```diff
@@ -649,6 +667,13 @@ fn add_plugin_deps(
     Ok(())
 }
 
+fn get_dynamic_search_path(path: &PathBuf) -> &Path {
```

Comment:
```suggestion
fn get_dynamic_search_path(path: &Path) -> &Path {
```

If nothing terribly needs it, we usually favor `&RefType` over `&OwnedType` for function arguments.